DEVELOPING.md

```diff
@@ -31,7 +23,9 @@ The command will produce the `revive` binary in the root of the project.
 
 If you want to develop a new rule, follow as an example the already existing rules in the [rule package](https://github.com/mgechev/revive/tree/master/rule).
 
-All rules should implement the following interface:
+## Writing a Custom Rule
```

Comment:
I think this header can be removed as the doc has `## Development of rules` above.